ServerRecording/pom.xml

```diff
@@ -45,32 +45,32 @@
 		<dependency>
 			<groupId>com.azure</groupId>
 			<artifactId>azure-identity</artifactId>
-			<version>1.3.5</version>
+			<version>1.5.4</version>
 		</dependency>
 		<dependency>
 			<groupId>com.azure</groupId>
 			<artifactId>azure-communication-callingserver</artifactId>
```

Comment:
Can you please update package reference to azure-communication-callautomation?  Package has been renamed.